src/helpers/index.ts

```diff
@@ -231,3 +231,14 @@ export function updateKeysFromIdsWithToken<TParams extends UpdateKeysFromsIdsDef
     return updateKeysFromIds(coin, ...rest);
   }
 }
+
+export function getEip1559Params(coin: string, maxFeePerGas: number, maxPriorityFeePerGas: number) {
+  // bsc and tbsc doesn't support EIP-1559
+  if (coin === 'bsc' || coin === 'tbsc') {
+    return undefined;
```

Comment:
you need to take it from user form, how is it building the tx with `undefined` value ?

Reply:
If `eip1559 : undefined`, in the sdk we set the default gas price. [Reference Link](https://github.com/BitGo/BitGoJS/blob/8b13b167463533a051143374b9fea8ccc1ad63b9/modules/abstract-eth/src/abstractEthLikeNewCoins.ts#L1308)